roles/servicetelemetry/tasks/main.yml

```diff
@@ -12,9 +12,9 @@
   set_fact:
     has_interconnect_api: "{{ True if 'interconnectedcloud.github.io' in api_groups else False }}"
 
-- name: Check if we have certmanager.k8s.io API
+- name: Check if we have cert-manager.io API
```

Comment:
I missed this the first time around and was wondering why my Interconnects weren't be created. This fixed it :)